tests/Kiota.Builder.Tests/Plugins/PluginAuthConfigurationTests.cs

```diff
@@ -46,15 +46,6 @@ public void AddCoverageOnUnsupportedException()
         Assert.NotEmpty(b.SupportedTypes);
     }
 
-    [Fact]
-    public void FromParametersThrowArgumentNullExceptionIfSecuritySchemeTypeHasNoValue()
-    {
-        Assert.Throws<ArgumentNullException>(() =>
-        {
-            _ = PluginAuthConfiguration.FromParameters(null, "reference");
-        });
-    }
```

Comment:
this test could be kept with an invalid operation exception instead

Reply:
Decided to keep the null check. Valid for a scenario where the method is used and there are no prior null checks.